lib/editor/components/ScheduleExceptionForm.js

```diff
@@ -218,29 +220,38 @@ export default class ScheduleExceptionForm extends Component<Props> {
             <ControlLabel>
               <small>On these dates*</small>
             </ControlLabel>
-            {ranges && ranges.map((range, index) => {
-              const startDateIndex = dates.findIndex(d => d === range.startDate)
-              const endDateIndex = dates.findIndex(d => d === range.endDate)
-              return (
-                <ExceptionDateRange
-                  index={index}
-                  key={uuidv4()}
-                  range={range}
-                  validationState={this._checkRangeValidation(startDateIndex, endDateIndex)}
-                  {...this.props}
-                />)
-            })}
-            {parsedDates && parsedDates.map((date, index) => {
-              const dateIndex = dates.findIndex(d => d === date)
-              return (
-                <ExceptionDate
-                  date={date}
-                  index={dateIndex}
-                  key={uuidv4()}
-                  validationState={this._checkValidation(`dates-${dateIndex}`)}
-                  {...this.props}
-                />)
-            })}
+            <FlipMove
+              enterAnimation='accordionVertical'
+              leaveAnimation='accordionVertical'
+              maintainContainerHeight
+              typeName={null}
+            >
+              {datesAndRangesBothActive && <h6 className='exception-subtitle'>Ranges</h6>}
```

Comment:
I would be in favor of always showing the respective header is there are any ranges/dates under it to help prevent headers from appearing/disappearing too much.

Reply:
The reason I was against this is because of the case where it says "on these dates" and then "dates" which I think is really confusing in an interface that is already quite complex

Reply:
We cannot use `<p>` because we rely on `h6` styles

Reply:
Can you move the applicable h6 styles to the `exception-subtitle` CSS?

Reply:
I don't understand the benefits of duplicating styles like that. Semantically it is correct that these headings are headings, and being level 6 makes it clear that they are not that important, although they are still headers.

Reply:
These are more similar to control labels than headings.